src/star-rating/index.js

```diff
@@ -0,0 +1,35 @@
+const simpleIcons = require("simple-icons");
+const starFilledSVG = require("./star-filled.svg.js");
+const starRegularSVG = require("./star-regular.svg.js");
+const starRatingComponentStyles = require("./index.style");
+
+const starRating = async (data = {}) => {
```

Comment:
```suggestion
const starRating = async ({ skill, rating }) => {
```

Something like this?

Reply:
done in https://github.com/harish-sethuraman/readme-components/commit/6edc8957e82aaaba15e112b02a4c6d1701498f39